src/services/file-upload.js

```diff
@@ -0,0 +1,48 @@
+const multer = require("multer");
+const path = require("path");
+const fs = require("fs");
+
+class FileUpload {
+  constructor() {
+    const storage = multer.diskStorage({
+      destination: (req, file, cb) => {
+        const assetsPath = path.join(__dirname, "..", "assets");
+        fs.existsSync(assetsPath) || fs.mkdirSync(assetsPath);
```

Comment:
do not use `sync` methods existsSync, mkdirSync

use fs.exist() and fs.mkdir()